src/main/java/games/strategy/triplea/attachments/PlayerAttachment.java

```diff
@@ -32,11 +32,11 @@ public static PlayerAttachment get(final PlayerID p) {
   }
 
   static PlayerAttachment get(final PlayerID p, final String nameOfAttachment) {
-    final PlayerAttachment rVal = p.getPlayerAttachment(); // (PlayerAttachment) p.getAttachment(nameOfAttachment);
-    if (rVal == null) {
+    final PlayerAttachment playerAttachment = p.getPlayerAttachment();
+    if (playerAttachment == null) {
       throw new IllegalStateException("No player attachment for:" + p.getName() + " with name:" + nameOfAttachment);
     }
-    return rVal;
+    return playerAttachment;
```

Comment:
Those getAttachment methods are all pretty similar, they could be unified and just call another method with a custom error message